lib/Sensors/THSensor.js

```diff
@@ -61,10 +61,10 @@ THSensor.prototype.getData = function (data) {
         this.humidity = parseInt(data.humidity) / 100;
         obj.humidity = this.humidity;
         newData = true;
-    }
-
+    }    
     if (data.pressure !== undefined && data.pressure !== null && data.pressure !== 0 && data.pressure !== '0') {
-        this.pressure = parseInt(data.pressure) / 100;
+        this.pressure = Math.round(parseInt(data.pressure) / 133.322);      // Pressure in mmHg
```

Comment:
You change the value? Was is broken before?

Reply:
No, everything worked. Here the conversion of units of measurement from hectopascals to mm Hg